cylc/flow/prerequisite.py

```diff
@@ -42,7 +42,7 @@ class Prerequisite:
 
     # Memory optimization - constrain possible attributes to this list.
     __slots__ = ["satisfied", "_all_satisfied",
-                 "target_point_strings", "start_point",
+                 "target_point_strings",
```

Comment:
Good spot, this appears to have been unused. Potentially a reasonable efficiency enhancement.

Reply:
Yes, I checked the spec etc.